client/task-list/tasks/PaymentGatewaySuggestions/components/Setup/Configure.js

```diff
@@ -34,7 +34,7 @@ export const validateFields = ( values, fields ) => {
 	return errors;
 };
 
-export const Connect = ( {
+export const Configure = ( {
```

Comment:
This rename seems to break a test in the `connect.js` test file, just requires a name update.